tests/test_polygons.py

```diff
@@ -1,6 +1,15 @@
 import pytest
 import tempfile
+import json
+import pathlib
+import logging
 from chap_core.geometry import Polygons
+from chap_core.geoutils import toposimplify, render
+
+
+TEST_PLOTS_FOLDER = pathlib.Path(__file__).parent / '__test_plots__'
```

Comment:
For now putting test renders inside a temporary ignored output folder, but maybe there is already another approach to this used elsewhere?

Reply:
Switched to path fixture inside an ignored folder